lib/tensorflow/session.rb

```diff
@@ -32,9 +32,9 @@ def run(inputs, outputs, targets)
     output_names, output_values = initialize_outputs(outputs)
     target_names = initialize_targets(targets)
 
-   	status = Tensorflow::TF_NewStatus()
-	  Tensorflow::TF_Run_wrapper(self.session, input_names, input_values, output_names, output_values, target_names, self.status)
-    raise ("Incorrect specifications passed.")  if Tensorflow::TF_GetCode(status) != Tensorflow::TF_OK
+   	status = Tensorflow::Internal::TF_NewStatus()
+	  Tensorflow::Internal::TF_Run_wrapper(self.session, input_names, input_values, output_names, output_values, target_names, self.status)
+    raise ("Incorrect specifications passed.")  if Tensorflow::Internal::TF_GetCode(status) != Tensorflow::Internal::TF_OK
```

Comment:
also in the section down here, sometimes there are leading tabs that make the indentation weird. Just 2 spaces per block level.